modules/nw-sriov-installing-operator.adoc

```diff
@@ -81,52 +78,77 @@ metadata:
   name: sriov-network-operator-subscription
   namespace: openshift-sriov-network-operator
 spec:
-  channel: <channel> <1>
+  channel: "<channel>" <1>
   name: sriov-network-operator
   source: redhat-operators <2>
   sourceNamespace: openshift-marketplace
 ----
-<1> Specify the value from you obtained in the previous step for the `.status.defaultChannel` parameter.
-<2> You must specify the `redhat-operators` value.
+<1> Specify the value of the `status.defaultChannel` field.
+<2> The value of `spec.source` must be `redhat-operators`.
 
-.. Create the Subscription object by running the following command:
+. To create the Subscription object, enter the following command:
 +
 ----
 $ oc create -f sriov-sub.yaml
 ----
 
-.. Change to the `openshift-sriov-network-operator` project:
+. To verify that the Operator is installed, enter the following command:
 +
 ----
-$ oc project openshift-sriov-network-operator
-
-Now using project "openshift-sriov-network-operator"
+$ oc get pods
 ----
+// What is the right verification step here?
```

Comment:
@zhaozhanqi  how did you verify if the operator is successfully installed via subscription?

Reply:
`oc get csv` can show the `PHASE`
```
oc get csv
NAME                                        DISPLAY                   VERSION              REPLACES                                    PHASE
sriov-network-operator.4.5.0-202005271737   SR-IOV Network Operator   4.5.0-202005271737   sriov-network-operator.4.5.0-202005220507   Succeeded
```

Reply:
@zhaozhanqi, perfect, thanks!

Reply:
@zshi-redhat, updated!